internal/errs/unwrap.go

```diff
@@ -13,5 +13,8 @@ func UnwrapExitCode(err error) int {
 		return eerr.ExitCode()
 	}
 
+	if err == nil {
```

Comment:
nit: maybe this should go to the top of the function?

Reply:
Yeah I agree, feels a bit awkward there.